harness/src/lib.rs

```diff
@@ -47,19 +47,26 @@ use slog::{Drain, Logger};
 /// Currently, this is a terminal log. It ensures it is only initialized once to prevent clobbering.
 // This is `pub` so that testing and benching functions can use it.
 // `#[cfg(test)]` doesn't work for benching.
-#[doc(hidden)]
```

Comment:
Why remove `doc(hidden)`? Seems like we don't want to make this public API

Reply:
Because harness is not a public library, will never be published to crates.io, doc hidden is unnecessary.